examples/v1.10.5/on-premise/additional/eco/docs.yaml

```diff
@@ -0,0 +1,50 @@
+###############################
+# DEFAULT 
+###############################
+enabled: true
+developer: false
+name: docs
+replicas: 1
+image:
+    name: pyengin/docs
+    version: 0.1.20221213.164753
+imagePullPolicy: IfNotPresent
+
+resources:
+    requests:
+        memory: "128Mi"
+        cpu: "25m"
+    limits:
+        memory: "256Mi"
+        cpu: "50m"
+
+imagePullSecrets: 
+  - name: {secret_name_for_credential}
+
+service:
+    type: ClusterIP
+    annotations:
+        nil: nil
+    ports:
+        - name: http
+          port: 80
+          targetPort: 80
+          protocol: TCP
+        - name: https
+          port: 443
+          targetPort: 80
+          protocol: TCP
+
+ingress:
+    enabled: false
+
+global:
+    namespace: {name_space}
+    serviceAccountName: {service_account_name}
+    common_env:
```

Comment:
The docs service is a static web page and requires no internal communication.
Therefore, it seems that you can remove all PROXY settings.

Reply:
Thank you for your feedback.
I agree with your feed, so I'll update this example.